jupyter_server/services/contents/fileio.py

```diff
@@ -264,7 +264,7 @@ def _get_os_path(self, path):
         ------
         404: if path is outside root
         """
-        self.log.debug("Reading path from disk: %s", path)
+        # self.log.debug("Reading path from disk: %s", path)
```

Comment:
I guess I'm not a fan of commenting out code without leaving a comment as to why.  Perhaps a comment could be something like:
```suggestion
        # This statement can cause excessive logging, uncomment if necessary when troubleshooting.
        # self.log.debug("Reading path from disk: %s", path)
```
or just delete the line altogether.